website/client/components/settings/subscription.vue

```diff
@@ -33,7 +33,7 @@
             span.noninteractive-button.btn-danger {{ $t('canceledSubscription') }}
             i.glyphicon.glyphicon-time
             |  {{ $t('subCanceled') }} &nbsp;
-            strong {{user.purchased.plan.dateTerminated | date}}
```

Comment:
can you remove the code for the code for the date filter below in this file since it's not used anymore? https://github.com/HabitRPG/habitica/pull/9583/files#diff-c55b95935ec2f85da360edbe1360852dL146